src/Mutator/InterfaceParentTrait.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © 2017 Volodimir Melko
```

Comment:
I think this is an autogenerated mistake. It should be as in other files - holder's name.
Not a big deal, but will be overridden by my auto-license-generating tool.

You can (and actually deserve ;)) to add your name as `@author` annotation, like e.g. symfony contributors do: https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php#L26